tests/test_cli/test_add/test_connection.py

```diff
@@ -511,3 +511,107 @@ def test_add_connection_latest_version(self):
         items_folders = os.listdir(items_path)
         item_name = "local"
         assert item_name in items_folders
+
+
+class TestAddConnectionMixedWhenNoLocalRegistryExists:
```

Comment:
why don't you use the AEA testing framework (e.g. see above `AEATestCaseEmpty`)? Anyway, it is good enough also without using it.

Reply:
Just reused existing test case. But yeah should be cleaned up eventually. Here we don't want the packages dir etc so makes sense to have a fresh setup anyway.